_config.yml

```diff
@@ -32,8 +32,10 @@ themes:          "https://themes.getbootstrap.com/"
 opencollective:  "https://opencollective.com/bootstrap"
 
 # Social
-# Default posts' author
-author:         "mdo"
+# Default author info
+author:
+  name:         "Mark Otto, Jacob Thornton, and Bootstrap contributors"
```

Comment:
Nearly every blog post has been written by me, save a few. Maybe we stick to me as the default author and the `getbootstrap` handle? We could then use that in the signoff of every post that I do.

Reply:
I will do it properly when I have some time. :)